.nancy-ignore

```diff
@@ -3,19 +3,3 @@
 # Affects all versions of archiver which is required by vault.
 # Taken from: https://github.com/giantswarm/opsctl/pull/1072/files#diff-bbe4a7fb12c43622bce7c6840c770e9995be614626a219942ca138403629cb69R1
 CVE-2019-10743 until=2021-10-17
-
-CVE-2021-20329 until=2022-06-30
```

Comment:
@giantswarm/team-honeybadger Do you want to keep these CVE ignores? They are all expired now